inference-engine/src/mkldnn_plugin/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -153,7 +153,11 @@ void MKLDNNFullyConnectedNode::execute(mkldnn::stream strm) {
 }
 
 bool MKLDNNFullyConnectedNode::canFuse(const MKLDNNNodePtr& node) const {
-    return canFuseSimpleOperation(node);
+    if (node->getType() == FakeQuantize) {
```

Comment:
Why do you update each node? Disabling binarization fuse should be default behavior of canFuseSimpleOperation(). Customization should be only inside BinaryConv.

Reply:
reverted